microceph/api/pool.go

```diff
@@ -18,6 +18,19 @@ import (
 var poolsCmd = rest.Endpoint{
 	Path: "pools-op",
```

Comment:
So the `pools-op` endpoint was created for special ops implemented for Ceph pools. There is no `/pools/` endpoint today, but for this operation, I feel a GET handler there would be more appropriate as when we do come up with the create/delete implementation it would fit nicely. Reservations against going ahead with `pools-op` because it will be LTS and we won't be able to deprecate it all-through 24.04's life.

Reply:
So should we leave `PUT` on `pools-op` or move that over to `/pools` as well?